applicationinsights.ts

```diff
@@ -225,9 +241,10 @@ class ApplicationInsights {
     }
 
     /**
-     * Disables verbose debug and warning logging
+     * Disables debug and warning logging for AppInsights itself.
+     * @returns {ApplicationInsights} this class
      */
-    public static disableConsoleLogging() {
+    public static disableVerboseLogging() {
```

Comment:
Should we still provide the old method as a deprecated copy that merely calls the new one?